src/modules/redesign/i18n/fr_FR/bewerbungen.json

```diff
@@ -0,0 +1,11 @@
+{
+	"accept": "Accept",
+	"alert": {
+		"accepted": "{username} was successfully accepted into the alliance!",
+		"declined": "The application of **{username}** was rejected!"
+	},
+	"decline": "Decline",
+	"rules": "Application rules",
+	"title": "Application: {n} | Applications: {n}",
+	"user": "Applicants"
+}
```

Comment:
translation?

Reply:
I have created all the files.
I translate them, sorry
Translation ok